### ISSUE TITLE
Missing documentation for accessibility support and compliance

### ISSUE THREAD

**Issue by missmatsuko** [ckeditor/ckeditor5#8171]
## 📝 Provide a description of requested docs changes

CKEditor 4 has several docs pages related to accessibility support and compliance:
- [Accessibility Support in CKEditor 4](https://ckeditor.com/docs/ckeditor4/latest/guide/dev_a11y.html)
- [CKEditor Section 508 Compliance](https://ckeditor.com/docs/ckeditor4/latest/guide/dev_section508.html)
- [CKEditor WCAG 2.0 Compliance](https://ckeditor.com/docs/ckeditor4/latest/guide/dev_wcag.html)

I wasn't able to find the equivalent information for CKEditor 5, except for maybe the [Keyboard support](https://ckeditor.com/docs/ckeditor5/latest/features/keyboard-support.html) page.

I'm not totally sure if CKEditor 5 is as up to date in terms of accessibility as CKEditor 4, but at least looking at the outstanding accessibility GitHub issues for both products, they seem fairly comparable.

_What is the purpose and what should be changed?_
Additional pages documenting accessibility support and compliance.

**Comment by Reinmar** [comment]
Hi!

Our target in CKEditor 5 is WCAG 2.1. We've designed CKEditor 5 with it in mind and done most of the work already. What we still need to do is:

*   Reviewing the current state in terms of WCAG 2.1 rules once again.
*   Creating a compliance summary for our documentation.
*   Resolving the issues that we'll find during the review.

We'd like to finish this ideally before Q2 2021.

**Comment by missmatsuko** [comment]
Thanks for the info, that sounds great! Looking forward to it! 🙂 I wonder if there could be placeholder page that has this info?

**Comment by o-alquimista** [comment]
How do we set an accessible label for the editor in CKEditor 5?

[CKEditor 4 documentation](https://ckeditor.com/docs/ckeditor4/latest/guide/dev_a11y.html#announcing-the-editor-on-the-page) tells us that it will announce whatever "title" the developer defined for the editor instance. I don't see the same feature in v5. The "placeholder" is the closest thing I've found, but it's not being announced to screen readers.

**Comment by o-alquimista** [comment]
@Reinmar How to change the "Rich Text Editor" label?

![Captura de tela de 2021-09-03 15-09-31](https://user-images.githubusercontent.com/25424263/132049056-3e0fbf0c-a4f6-4e73-8e87-90f0b8977a9d.png)

**Comment by mvernier-fuze** [comment]
@Reinmar any updates on this issue? My team is looking to upgrade from CKEditor 4 to CKEditor 5. WCAG 2.1 compliance is a requirement for our clients. We previously linked to version 4 documentation in our VPATs. Thanks!

**Comment by jannakha-copy-01** [comment]
+1 for this issue

CKEditor team - you don't have to be 100% compliant at the moment, accessibility is work in progress and you are very committed to it too!

suggested solution for now will be adding accessibility section to documentation with commitment statement, and a link to github issue queue tagged with **domain:accessibility**

you can update accessibility section as you progress of the issues, and later add testing outcomes!

**Comment by Witoso** [comment]
Sorry for not updating the issue, we are actually in the process of auditing and preparing such documents, led by @Comandeer, @DawidKossowski, and @oleq. Stay tuned!

**Comment by oleq** [comment]
For anyone interested: a new guide has landed in the project's documentation focusing specifically on editor accessibility and compliance with the standards (https://ckeditor.com/docs/ckeditor5/latest/features/accessibility.html). I hope this helps you and answers all your questions. Thank you for your feedback!

**Comment by gcamacho079** [comment]
@oleq @Comandeer @DawidKossowski appreciate your team for your efforts in putting this together! It’s so valuable to have this documentation and an accompanying VPAT.
